src/ucp/core/ucp_mm.c

```diff
@@ -366,6 +366,15 @@ void ucp_memh_cleanup(ucp_context_h context, ucp_mem_h memh)
     }
 }
 
+static void
+ucp_memh_apply_reg_nb_flags(ucp_context_h context, ucs_memory_type_t mem_type,
```

Comment:
maybe no need a separate function?
@brminich WDYT?

Reply:
yes, we can avoid that